src/entries/popup/pages/home/Tokens.tsx

```diff
@@ -285,7 +301,11 @@ export function Tokens() {
                 style={{ height: size, y: start }}
               >
                 {pinned && <TokenMarkedHighlighter />}
-                <TokenRow token={token} testId={`coin-row-item-${index}`} />
+                <TokenRow
+                  containerRef={containerRef}
```

Comment:
My guess is that you're passing this all the way down to the context menus so that if we use a shortcut, we can simulate a click on the main container to close the context menu?

Do you think there'd be negative performance implications in just using this hook in the context menu itself rather than prop drilling it down two levels?